Node.js/GettingStarted/startup_prereq.sh

```diff
@@ -0,0 +1,17 @@
+cd $(dirname $0)
+sudo apt-get update && echo 'y' | sudo apt-get upgrade
```

Comment:
Yes, it's Ubuntu only. The Ubuntu distros most of the time are very old, and we need the apt-get update to apply latest patches. For our own SDKs to function.